crates/contracts/src/lib.rs

```diff
@@ -3,9 +3,9 @@
 #![cfg_attr(not(test), warn(unused_crate_dependencies))]
 #![cfg_attr(docsrs, feature(doc_cfg))]
 
-use alloy::primitives::{Address, address};
+use alloy_primitives::{Address, address};
 
-pub const MULTICALL_ADDRESS: Address = alloy::providers::MULTICALL3_ADDRESS;
+pub const MULTICALL_ADDRESS: Address = address!("0xcA11bde05977b3631167028862bE2a173976CA11");
```

Comment:
this is the correct one, can we copy past
```suggestion
/// Default address for the Multicall3 contract on most chains. See: <https://github.com/mds1/multicall>
pub const MULTICALL_ADDRESS: Address = address!("0xcA11bde05977b3631167028862bE2a173976CA11");
```